kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -226,6 +230,20 @@ def print_failure_info(proof: Proof, kcfg_explore: KCFGExplore) -> list[str]:
         raise ValueError('Unknown proof type.')
 
 
+def print_model(node: KCFG.Node, kcfg_explore: KCFGExplore) -> list[str]:
+    res_lines: list[str] = []
+    result_subst = kcfg_explore.cterm_get_model(node.cterm)
+    if type(result_subst) is Subst:
+        res_lines.append('  Model:')
+        for var, term in result_subst.to_dict().items():
+            term_kast = KInner.from_dict(term)
+            res_lines.append(f'    {var} = {kcfg_explore.kprint.pretty_print(term_kast)}')
+    else:
+        res_lines.append('  Failed to generate a model.')
+
+    return res_lines
```

Comment:
We rewrote the `print_failure_info` function in pyk to separate the printing from the RPC calls here: https://github.com/runtimeverification/pyk/commit/6c28b6613ea058a46fd4f838982a88bc2e957863
Possibly we would want to do something similar here, since it makes testing cleaner. I realized we haven't updated kevm to use that function yet and we still just use the kevm version, so actually we can probably just refactor this when we make those changes later.

Reply:
Thanks Noah! I can probably try integrating the new `failure_info` function into kevm to include it in this PR, if this works--will update you on that tomorrow.